Sources/CodegenCLI/Commands/Initialize.swift

```diff
@@ -15,7 +15,26 @@ public struct Initialize: ParsableCommand {
     name: [.long, .customShort("n")],
     help: "Name used to scope the generated schema type files."
   )
-  var schemaName: String = ""
+  var schemaName: String
+
+  @Option(
+    name: [.long, .customShort("m")],
+    help: """
+      How to package the schema types for dependency management. Possible types: \
+      \(ModuleTypeExpressibleByArgument.allValueStrings.joined(separator: ", ")).
```

Comment:
I'm joining them here with `,` because `allValueStrings` printed output contains the array braces `[]`.